### DISCUSSION TITLE
guide/install/docker

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#212] [Announcements]
# guide/install/docker

See the log output for the admin's info: Release version docker-cli docker-compose Offline download with aria2 If you want to use aria2 to offline download, we recommend you to ...

https://alist.nn.ci/guide/install/docker.html

**Comment by printelife123** [comment]
Why can I only get version 3.10.1 through docker pull? Deleting “registry-mirrors” doesn't work either

**Comment by AspadaX** [comment]
How to mount existing hard drives on the host machine? It seems that the docker version can now only access the docker itself...
